### PR TITLE
Hangar Lockdown Fixes

### PR DESCRIPTION
**Pull request by Ben10083**
Adds missing lockdown shutters for Hangar Lockdown (such as windows, some of the doors, etc)

### CODE REVIEW

**Comment by Ben10083** [comment]
!review

**Comment by Ben10083** [comment]
!bugfix

**Review by hazelrat — APPROVED** [review]

**Review by FluffyGhoster — APPROVED** [review]

File: html/changelogs/Ben10083 - Lockdown Fix.yml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+# Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, this gets changed to [] after reading.  Just remove the brackets when you add new shit.
+# Please surround your changes in  double quotes ("). It works without them, but if you use certain characters it screws up compiling. The quotes will not show up in the changelog.
+changes:
+  - bugfix: "Adds several missing hangar lockdown doors."
```
Comment (SleepyGemmy) [inline html/changelogs/Ben10083 - Lockdown Fix.yml:58]:
```suggestion
  - bugfix: "Adds several missing hangar lockdown blast doors."
```

**Review by SleepyGemmy — CHANGES_REQUESTED** [review]
misplaced blast door.
image

**Review by SleepyGemmy — APPROVED** [review]

**Pull request merged**